{{cookiecutter.project_slug}}/docs/conf.py

```diff
@@ -14,11 +14,20 @@
 import sys
 import django
 
-{% if cookiecutter.use_docker == 'y' %}
-sys.path.insert(0, os.path.abspath("/app"))
-os.environ.setdefault("DATABASE_URL", "")
-{% else %}
-sys.path.insert(0, os.path.abspath(".."))
+if os.getenv("READTHEDOCS", default=False) == "True":
+    sys.path.insert(0, os.path.abspath(".."))
+    os.environ["DJANGO_READ_DOT_ENV_FILE"] = "True"
+    {%- if cookiecutter.use_celery == 'y' %}
+    os.environ["CELERY_BROKER_URL"] = os.getenv("REDIS_URL", "redis://redis:6379")
+    {%- endif %}
+    os.environ["USE_DOCKER"] = "no"
+    master_doc = "_source/index"
+else:
+{%- if cookiecutter.use_docker == 'y' %}
+    sys.path.insert(0, os.path.abspath("/app"))
+    os.environ.setdefault("DATABASE_URL", "")
```

Comment:
This environbment variable needs to be set in RTD at the moment... I'm think, would it make sense to have the steps required documented?

These are the steps I had to take manually, which I don't think can be automated at the moment:

1. Go to readthedocs.org and login/create and account
2. Add the Github repository
3. Setup the environemnt variable `DATABASE_URL=sqlite:///database.db`
4. Trigger a build

We could then link to a few handy advanced settings from RTD like [pull request previews](https://docs.readthedocs.io/en/latest/guides/autobuild-docs-for-pull-requests.html#autobuild-documentation-for-pull-requests) or versioned documentation.

Reply:
Yes, we should definitely document the manual steps in the docs. I can add that in. Although I'm not sure if the `DATABASE_URL=sqlite:///database.db` is necessary... 

I think referring to our previous conversation to how docs are built, we should automatically create that database and even perhaps rename it to `.readthedocs.db` and leave the fact that we manually changed the DB in the docs. Which reminds me, did it create that sqlite db?

Reply:
Ideally, that step shouldn't be there, but for the moment we have this other issue that (I think) you were planning to fix in the other PR... As it's out of scope here, I was suggesting to document it until it's fixed.